clustering/server/src/main/java/org/wildfly/clustering/server/dispatcher/ChannelCommandDispatcherFactory.java

```diff
@@ -132,13 +133,8 @@ public Object handle(Message message) throws Exception {
                 if (context == null) return NoSuchService.INSTANCE;
```

Comment:
I'm not sure we allow such statement...

Reply:
On the contrary, this can easily be the case when using ForkChannels.

Reply:
sorry, what I mean is that I'm not sure we allow to have if statement without curly braces on a single line :)

Reply:
Oh - I see. :)
A quick search of the wildfly code base reveals 308 uses of 1 liners for "simple if" statements, and 183 in wildfly-core.